scripts/!mods_preload/msu_aa.nut

```diff
@@ -1,21 +1,12 @@
-local gt = this.getroottable();
+::MSU <- {};
+::MSU.Version <- "1.0.0-alpha";
+::MSU.ID <- "mod_MSU";
+::MSU.Name <- "Modding Standards & Utilities";
 
-local modID = "mod_MSU";
-::mods_registerMod(modID, 1.0, "Modding Standards and Utils 0.5.3");
+::MSU.VanillaID <- "vanilla";
```

Comment:
we should probs also have a vanilla name and use that

Reply:
Sure. Alternatively capitalize the ID as our systems handle nameless mods with their ids. But I'm ok if we wanna keep id lowercase and have name separately.

Reply:
On second thought, we are passing "Vanilla" as the name of the vanilla mod when registering it, so I don't think we need a separate VanillaName key here.